terminus-ui/table/src/table.component.scss

```diff
@@ -21,40 +21,12 @@ $ts-row-hover: rgba(color(utility, xlight), .5);
 .ts-table {
   @include reset;
   @include typography;
```

Comment:
OK, spent a while trying to figure out an odd scrolling issue where the pinning just 'stops':

![Kapture 2019-08-13 at 16 28 13](https://user-images.githubusercontent.com/270193/62975081-8d771f00-bde7-11e9-8ba7-51fd13cedbbc.gif)

Notice the stickiness ends right when the row ends (see the background color ending). The browser handles this incorrect layout fine on its own when there is no sticky trickery.

It looks like adding `display: inline-block` to `.ts-table` fixes this

Reply:
This breaks the stickiness.

Reply:
Can you clarify this? It was definitely working for me